src/index.html

```diff
@@ -12,7 +12,40 @@
       href="styles/index.scss"
     />
```

Comment:
Link to a stylesheet with a `.scss` extension won't work properly in a browser. You should compile SCSS to CSS and link to the compiled `.css` file instead.